models/articles.models.js

```diff
@@ -8,4 +10,23 @@ exports.selectArticleById = (article_id) => {
         return rows.rows[0]
     })
 
-};
\ No newline at end of file
+
+};
+
+exports.selectAllArticles = () => {
+    
+
+    return db.query(`SELECT articles.*, COUNT(comments.article_id) AS comment_count
+    FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id
+    GROUP BY articles.article_id
+    ORDER BY articles.created_at DESC;`)
+        .then((articles) => {
+            let filteredArticles = articles.rows;
+            filteredArticles = filteredArticles.map((article) => {
+                delete article.body;
+                return article;
+            });
+            return filteredArticles;
+        })
+  };
```

Comment:
Is there a reason you are manually removing the body property separately here and not letting the query handle this for you?